CHANGELOG.rst

```diff
@@ -7,6 +7,11 @@ Rasa Change Log
 All notable changes to this project will be documented in this file.
 This project adheres to `Semantic Versioning`_ starting with version 1.0.
 
+[Unreleased 1.3.5]
```

Comment:
I am so sorry, but could you please update this one more time? We had a busy week and cut a couple of bug fixes releases in the mean time 🙈

Reply:
Alright, took a little work, but this PR is at least slated for 1.3.7.  Let me know if there's anything else needed to merge this.